src/index.js

```diff
@@ -24,50 +24,78 @@ const editTodo = (i, descr) => {
   window.location.reload();
 };
 
+const clearCompleted = () => {
+  const newItems = [];
+
+  todo.getAll().forEach((item) => {
+    if (!item.completed) {
+      newItems.push(item);
+    }
+  });
+
+  todo.data = newItems;
+  todo.updateIndexes();
+
+  localStorage.setItem('mydata', JSON.stringify(todo.getAll()));
+  window.location.reload();
+};
+
```

Comment:
- [OPTIONAL] Good job implementing the required change by the previous reviewer. But, there's one bug I noticed while testing your app for proper behavior. All completed tasks get deleted from the browser, but remain in the local storage when you refresh the page without clicking on the clear completed button. Clear completed button works as expected but I believe is worth it to look into this bug and fix it. I marked this optional because the previous reviewer did not notice this and as a result did not call your attention to it at first place, otherwise, I would have made it required. I would recommend you to attend to it and fix. Take a look.
  
[app-behavior.webm](https://user-images.githubusercontent.com/26597432/222884083-5ee8926e-f1cd-4ccf-81f9-b2b37e8caebe.webm)